docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -1314,6 +1314,11 @@ PROXY_URL = socks://127.0.0.1:1080
 PROXY_HOSTS = *.github.com
 ```
 
+## Sync (`sync`)
+
+- `LOCK_SERVICE_TYPE`: **memory**: Lock service type, could be `memory` or `redis`
+- `LOCK_SERVICE_CONN_STR`: **\<empty\>**: Ignored when `LOCK_SERVICE_TYPE` is `memory` type, for `redis`, it likes `addrs=127.0.0.1:6379 db=0`
```

Comment:
shouldn't this use the same connection string format which the other redis types are using?